01week/datatypes.js

```diff
@@ -47,6 +58,12 @@ var u = undefined;
 var p = 1;
 var a = "b";
 
```

Comment:
You created a great example of each data type, but could you create a program that prints that data type instead? Such as var f = (10 > 9)